src/LRS.js

```diff
@@ -887,6 +888,42 @@ TinCan client library
         },
 
         /**
+        Assigns attachment content to the correct attachment to create a StatementsResult object that is sent
+        to the callback of queryStatements()
+
+        @method _assignAttachmentContent
+        @private
+        @param {Array} [stmts] Array of TinCan Statement JSON objects
+        @param {Array} [headers] Array containing section headers from the HTTP response
+        @param {String} [content] The content to place into its attachment
+        @return {Array} Array of TinCan Statement JSON objects with correctly assigned attachment content
+        */
+        _assignAttachmentContent: function (stmts, headers, content) {
+            var hashToMatch,
+                i,
+                j;
+
+            for (i = 0; i < headers.length; i += 1) {
+                if (headers[i].indexOf("X-Experience-API-Hash") >= 0) {
+                    hashToMatch = headers[i].split(":")[1].trim();
+                }
+            }
+
+            for (i = 0; i < stmts.statements.length; i += 1) {
+                if (stmts.statements[i].hasOwnProperty("attachments") && stmts.statements[i].attachments !== null) {
```

Comment:
@brianjmiller At this point they're just JSON, and then I turn them into to `TinCan.Statement` objects after this function returns.